Author: Pino Toscano <pino@debian.org>
Description: initialize PSOutputDev::fontFileNameLen and PSOutputDev::psFileNames
 Avoid crashing in ~PSOutputDev when the PSOutputDev instance is not "ok".
Applied-Upstream: not-needed
Last-Update: 2013-01-31
Bug-Debian: http://bugs.debian.org/699421

--- a/poppler/PSOutputDev.cc
+++ b/poppler/PSOutputDev.cc
@@ -1012,6 +1012,7 @@ PSOutputDev::PSOutputDev(const char *fil
   fontIDs = NULL;
   fontFileIDs = NULL;
   fontFileNames = NULL;
+  fontFileNameLen = 0;
   font8Info = NULL;
   font16Enc = NULL;
   imgIDs = NULL;
@@ -1022,6 +1023,7 @@ PSOutputDev::PSOutputDev(const char *fil
   haveTextClip = gFalse;
   haveCSPattern = gFalse;
   t3String = NULL;
+  psFileNames = NULL;
 
   forceRasterize = forceRasterizeA;
 
@@ -1077,6 +1079,7 @@ PSOutputDev::PSOutputDev(PSOutputFunc ou
   fontIDs = NULL;
   fontFileIDs = NULL;
   fontFileNames = NULL;
+  fontFileNameLen = 0;
   font8Info = NULL;
   font16Enc = NULL;
   imgIDs = NULL;
@@ -1087,6 +1090,7 @@ PSOutputDev::PSOutputDev(PSOutputFunc ou
   haveTextClip = gFalse;
   haveCSPattern = gFalse;
   t3String = NULL;
+  psFileNames = NULL;
 
   forceRasterize = forceRasterizeA;
 
